streamwebs_frontend/streamwebs_frontend/settings.py

```diff
@@ -129,6 +129,12 @@
 
 STATIC_URL = '/static/'
 
+# Temporary list of schools
+SCHOOL_CHOICES = (
```

Comment:
Leave this empty - if you need a list for a test, set the value in the test itself.